easy-ui-react/src/Button/Button.tsx

```diff
@@ -41,6 +41,67 @@ export type ButtonProps = AriaButtonProps & {
   href?: string;
 };
 
+/**
+ * Allow users to preform an action or to navigate to another page.
+ *
+ * @remarks
+ * Use to call attention to a place in the UI where the user needs to do
```

Comment:
**Nit:** I'm not sure _needs_ is the right word since often it's by choice but I don't have an alternate suggestion

Reply:
updated sentence, open to other suggestions/ideas